projects/igniteui-angular/src/lib/services/excel/excel-files.ts

```diff
@@ -85,42 +87,52 @@ export class WorksheetFile implements IExcelFile {
             this.dimension = 'A1';
             done('', sheetData);
         } else {
-            sheetData += '<sheetData>';
+            const isHierarchicalGrid = worksheetData.data[0].type === ExportRecordType.HierarchicalGridRecord;
+
             const height =  worksheetData.options.rowHeight;
-            this.rowHeight = height ? ' ht="' + height + '" customHeight="1"' : '';
-            sheetData += `<row r="1"${this.rowHeight}>`;
+            const rowStyle = isHierarchicalGrid ? ' s="3"' : '';
+            this.rowHeight = height ? ` ht="${height}" customHeight="1"` : '';
```

Comment:
We could probably extract the setting rowHeight logic in a method (I saw this used elsewhere).